scripts/lib/clean_unused_caches.py

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env python3
+import argparse
```

Comment:
You need to update things that call this script when renaming it.

Reply:
Done!